NodeManager.cpp

```diff
@@ -1955,6 +2095,14 @@ int NodeManager::registerSensor(int sensor_type, int pin, int child_id) {
       return registerSensor(new SensorDHT(child_id,pin,dht,SensorDHT::HUMIDITY,dht_type));
     }
   #endif
+  #if MODULE_AM2320 == 1
+    else if (sensor_type == SENSOR_AM2320) {
+      //AM2320* th = new AM2320();
```

Comment:
Sorry i pulled request too early, it's still in testing...
To answer your question, I had readings problem when calling the AM2320 object here.
I can insist with this approach to find a solution.